spec/scry/workspace_spec.cr

```diff
@@ -1,4 +1,5 @@
 require "../spec_helper"
+require "tempfile"
```

Comment:
Note: Tempfile will be removed in next Crystal release

Reply:
Yeah I made a comment in the specs that it will need to be changed w/ v0.27.0